app/test/AstroMath_test.js

```diff
@@ -0,0 +1,86 @@
+import 'jsdom-global/register';
+import AstroMath from '../src/js/AstroMath';
+import { expect } from "chai";
+
+// global class AstroMath variable
+let testMath = new AstroMath("mars");
+
+describe('Testing AstroMath Functions', function() {
+
+  // testing AstroMath created a class object
+  it('Testing AstroMath Object', function() {
+    // add an assertion
+    
+    expect(testMath).to.be.an('Object');
+  })
+
+  // testing getMajorRadius function
+  it('Testing getMajorRadius: Mars should be 3396190', function() {
+    // add an assertion
+    let majorRadius = testMath.getMajorRadius();
+    expect(majorRadius).to.equal(3396190);
+  })
+
+  // testing getMinorRadius function
+  it('Testing getMinorRadius: Mars should be 3376200', function() {
+    // add an assertion
+    let majorRadius = testMath.getMinorRadius();
+    expect(majorRadius).to.equal(3376200);
+  })
+
+  // testing toRadians function
+  it('Testing toRadians: 360 degrees should be 6.283185307179586', function() {
+    // add an assertion
+    let majorRadius = testMath.toRadians(360);
+    expect(majorRadius).to.equal(6.283185307179586);
```

Comment:
Is there a way we can expect the answer to be near instead of having all of the decimal places?